app/assets/stylesheets/v3/layout_components/pre-footer.scss

```diff
@@ -0,0 +1,102 @@
+.pre-footer {
+  @include grid-column(12,0);
+  background: $white;
+  height: $side-nav-height;
+  top: $top-bar-height;
+  z-index: 9000;
+
+  @include breakpoint(medium down) {
+    display: none;
+  }
+
+  .logo {
```

Comment:
did we ever figure out what happened to the logo? Also, do we need this styling still?

Reply:
I hid the logo in the top bar when I moved it into the sidebar (It basically bumped down from the very top to the second top nav)
Since this top bar doesn't use API to display the logo, we will likely just need to add same markup that is used to add the logo in API.

Reply:
Thanks @DaveVanVliet